pkg/controllers/dynakube/extension/secret.go

```diff
@@ -0,0 +1,83 @@
+package extension
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dttoken"
+	k8ssecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	eecTokenKey         = "eec-token"
+	eecTokenValuePrefix = "EEC dt0x01"
+	secretSuffix        = "-extensions-token"
+)
+
+func reconcileSecret(ctx context.Context, dk *dynakube.DynaKube, kubeClient client.Client, apiReader client.Reader) error {
+	log.Info("reconciling secret" + getSecretName(dk.Name))
```

Comment:
may be we need add a space here ^

Reply:
Fixed [here](https://github.com/Dynatrace/dynatrace-operator/pull/3485/commits/24ace1ad5534ddee99b63515d3e7b766683561bc)